ycms/cms/templates/_base.html

```diff
@@ -111,17 +111,19 @@
         <li class="hidden mb-4" id="timetravel-stop">
             <a href="?" class="justify-center"><i icon-name="timer-off"></i> {% translate "Stop Timetravel" %}</a>
         </li>
-        <button id="theme-toggle"
-                data-tooltip-target="tooltip-right"
-                data-tooltip-placement="right"
-                type="button"
-                class="text-gray-500 dark:text-gray-400 hover:bg-gray-100 dark:hover:bg-gray-600 focus:outline-none focus:ring-4 focus:ring-gray-200 dark:focus:ring-gray-800 rounded-lg text-sm p-2.5 mb-4">
-            {% if is_dark_theme %}
-                <i icon-name="sun"></i>
-            {% else %}
-                <i icon-name="moon"></i>
-            {% endif %}
-        </button>
+        <a href="{% url 'change-theme' %}">
+            <button id="theme-toggle"
+                    data-tooltip-target="tooltip-right"
+                    data-tooltip-placement="right"
+                    type="button"
+                    class="text-gray-500 dark:text-gray-400 hover:bg-gray-100 dark:hover:bg-gray-600 focus:outline-none focus:ring-4 focus:ring-gray-200 dark:focus:ring-gray-800 rounded-lg text-sm p-2.5 mb-4">
+                {% if is_dark_theme %}
+                    <i icon-name="sun"></i>
+                {% else %}
+                    <i icon-name="moon"></i>
+                {% endif %}
+            </button>
+        </a>
```

Comment:
I've been meaning to ask, because I've seen you use the `button`-inside-`a` pattern a couple of times now, is there a reason to not just do this? 😄 
```suggestion
        <a href="{% url 'change-theme' %}"
              data-tooltip-target="tooltip-right"
              data-tooltip-placement="right"
              class="inline-block text-gray-500 dark:text-gray-400 hover:bg-gray-100 dark:hover:bg-gray-600 focus:outline-none focus:ring-4 focus:ring-gray-200 dark:focus:ring-gray-800 rounded-lg text-sm p-2.5 mb-4">
          {% if is_dark_theme %}
              <i icon-name="sun"></i>
          {% else %}
              <i icon-name="moon"></i>
          {% endif %}
        </a>
```